test/cypress/support/command/command.js

```diff
@@ -12,6 +14,24 @@ Cypress.Commands.add('getByTestId', (selector, ...args) => cy.get(`[${TEST_ID_KE
  */
 Cypress.Commands.add('getById', (selector, ...args) => cy.get(`[id=${selector}]`, ...args));
 
+/**
+ *  Get input elements by name
+ *  @param selector - The name of the DOM elements
+ */
+Cypress.Commands.add('getInputByName', (name, ...args) => cy.get(`input[name=${escapeRegExp(name)}]`, ...args));
+
+/**
+ *  Get Typeahead Option by value
+ *  @param value - The the exact Option text displayed in the UI
+ */
+Cypress.Commands.add('getTypeaheadOption', (value, ...args) => cy.get('.dropdown-item', ...args).contains(value));
```

Comment:
then `getTypeaheadOptionByValue` is probably clearer

Reply:
Done